Galaxy_Workflow_MIRACUM_main.ga

```diff
@@ -1,2760 +1,2953 @@
-            "id": 14,
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/qualimap_bamqc/qualimap_bamqc/2.2.2c+galaxy1",
             "tool_shed_repository": {
+                "changeset_revision": "4a89c6f84425",
+                "name": "qualimap_bamqc",
                 "owner": "iuc",
-                "changeset_revision": "3d93dd18d9f8",
-                "name": "multiqc",
                 "tool_shed": "toolshed.g2.bx.psu.edu"
             },
-            "uuid": "37517fa1-c84d-43f9-8786-ce010e78c8d1",
-            "errors": null,
-            "name": "MultiQC",
-            "post_job_actions": {
-                "HideDatasetActionplots": {
-                    "output_name": "plots",
-                    "action_type": "HideDatasetAction",
-                    "action_arguments": {}
-                },
-                "RenameDatasetActionhtml_report": {
-                    "output_name": "html_report",
-                    "action_type": "RenameDatasetAction",
-                    "action_arguments": {
-                        "newname": "QC report for processed sequencing data"
-                    }
+            "tool_state": "{\"duplicate_skipping\": [\"0\"], \"input1\": {\"__class__\": \"ConnectedValue\"}, \"per_base_coverage\": \"false\", \"plot_specific\": {\"n_bins\": \"1000\", \"paint_chromosome_limits\": \"true\", \"genome_gc_distr\": null, \"homopolymer_size\": \"3\"}, \"stats_regions\": {\"region_select\": \"custom_regions\", \"__current_case__\": 1, \"regions\": {\"__class__\": \"ConnectedValue\"}, \"outside_stats\": \"false\"}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "2.2.2c+galaxy1",
+            "type": "tool",
+            "uuid": "1f5e90ff-b328-460f-ac74-ecde4d9973ce",
+            "workflow_outputs": [
+                {
+                    "label": "QualiMap BamQC report Tumor",
+                    "output_name": "output_html",
+                    "uuid": "64104618-6fac-46e1-9def-a7d19fdbbb43"
                 },
-                "HideDatasetActionstats": {
-                    "output_name": "stats",
-                    "action_type": "HideDatasetAction",
-                    "action_arguments": {}
+                {
+                    "label": null,
+                    "output_name": "raw_data",
+                    "uuid": "187f51e9-8912-41c8-8dea-55d75d160a2f"
```

Comment:
There is one last undefined label at this step.

Reply:
Thank you for pointing that out!